plausible/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: plausible
+icon: https://svgshare.com/i/y30.svg
+category: developer
+name: Plausible Analytics
+version: "2.1.1"
+tagline: Easy to use and privacy-friendly Google Analytics alternative
+description: >-
+  Plausible is intuitive, lightweight and open source web analytics.
+  No cookies and fully compliant with GDPR, CCPA and PECR.
+  Made and hosted in the EU, powered by European-owned cloud infrastructure.
+releaseNotes: ""
+developer:  Plausible Community Edition
+website: https://plausible.io/
+repo: https://github.com/plausible/community-edition/
+support: https://github.com/plausible/community-edition/issues
+port: 9091
```

This port is already taken by the Transmission app, so if a user has this installed, then their Plausible installation will break when they attempt installation. You can choose an arbitrary port here and don't have to worry about what Plausible itself uses since our app_proxy container will handle it.

Port 9092 is free.
```suggestion
port: 9092
```